patterns/2-structured/core-team.md

```diff
@@ -97,6 +97,7 @@ The core team fills those gaps and greases the wheels so that the contribution e
 
 * **Nike** implemented this pattern to manage the InnerSource effort around its reusable CI/CD pipelines.
 * **WellSky** established a Core Team for a key project. This allowed them to scale their InnerSource contributions to that project significantly - see [Wide-Scaled InnerSource with a Core Team](https://www.youtube.com/watch?v=kgxexjYdhIc).
+* **BBVA AI Factory** implemented this pattern for the Mercury project. It applies an InnerSource strategy to foster contribution and reuse of data science code - see [Mercury: Scaling Data Science reusability at BBVA](https://www.bbvaaifactory.com/mercury-acelerando-la-reutilizacion-en-ciencia-de-datos-dentro-de-bbva/)
```

Comment:
```suggestion
* **BBVA AI Factory** implemented this pattern as part of an InnerSource strategy to foster contribution to and reuse of data science code - see [Mercury: Scaling Data Science reusability at BBVA](https://www.bbvaaifactory.com/mercury-acelerando-la-reutilizacion-en-ciencia-de-datos-dentro-de-bbva/).
```

Reply:
@rahermur just like for the website story, I rewrote this sentence slightly so that the name Mercury is less prominent. I think it will be easier to understand for the readers that way.

If you are fine with this change, I am happy to merge the PR.

Reply:
It is great. Thanks